app-backend/migrations/src_migrations/main/scala/24.scala

```diff
@@ -0,0 +1,18 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M24 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(24)(List(
+    sqlu"""
+CREATE TABLE bands (
+  id UUID PRIMARY KEY NOT NULL,
+  image_id UUID REFERENCES images(id) NOT NULL,
+  name VARCHAR(255) NOT NULL,
+  number INTEGER NOT NULL,
+  wavelength INTEGER[] NOT NULL
+);
+
+ALTER TABLE images DROP COLUMN bands;
```

Comment:
No, the existing information isn't useful. We couldn't create Bands from the existing info in bands without considerable effort, and the importer will take care of it for us anyway (at least once I get that PR up)